samples/Samples.HttpClient.MissingDotnetTracerHomeEnv/Properties/launchSettings.json

```diff
@@ -0,0 +1,19 @@
+{
+    "profiles": {
+      "Samples.HttpClient.MissingDotnetTracerHomeEnv": {
+        "commandName": "Project",
+        "environmentVariables": {
+          "COR_ENABLE_PROFILING": "1",
+          "COR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
+          "COR_PROFILER_PATH": "$(ProjectDir)$(OutputPath)profiler-lib\\Datadog.Trace.ClrProfiler.Native.dll",
+  
+          "CORECLR_ENABLE_PROFILING": "1",
+          "CORECLR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
+          "CORECLR_PROFILER_PATH": "$(ProjectDir)$(OutputPath)profiler-lib\\Datadog.Trace.ClrProfiler.Native.dll",
+  
+          "DD_INTEGRATIONS": "$(ProjectDir)$(OutputPath)profiler-lib\\integrations.json"
```

Comment:
This very simple test program uses `DD_INTEGRATIONS` and doesn't use `DD_DOTNET_TRACER_HOME` to simulate a user who has upgraded the profiler without updating the environment variables.